ydb/core/kqp/provider/yql_kikimr_datasink.cpp

```diff
@@ -864,7 +864,7 @@ class TKikimrDataSink : public TDataProviderBase
             return false;
         }
 
-        if (tableDesc.Metadata->Kind == EKikimrTableKind::Olap && mode != "replace" && mode != "drop" && mode != "drop_if_exists") {
+        if (tableDesc.Metadata->Kind == EKikimrTableKind::Olap && mode != "replace" && mode != "drop" && mode != "drop_if_exists" && mode != "insert_abort" && mode != "update" && mode != "upsert" && mode != "delete" && mode != "update_on" && mode != "delete_on") {
```

Comment:
You can get rid of this "if" statement. If something doesn't work here then it's a bug that should be fixed.